catroid/src/main/java/org/catrobat/catroid/content/bricks/SetInstrumentBrick.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2020 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.content.bricks;
+
+import android.content.Context;
+import android.view.View;
+
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.common.Nameable;
+import org.catrobat.catroid.content.Sprite;
+import org.catrobat.catroid.content.actions.ScriptSequenceAction;
+import org.catrobat.catroid.content.bricks.brickspinner.BrickSpinner;
+import org.catrobat.catroid.pocketmusic.note.MusicalInstrument;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import androidx.annotation.Nullable;
+
+public class SetInstrumentBrick extends BrickBaseType {
+
+	public enum PickableMusicalInstrument implements Nameable {
```

Comment:
please avoid creating nested classes in Bricks, 
* move `PickableMusicalInstrument` to `org.catrobat.catroid.content.bricks.brickspinner;` and make it class instead of enum
* let this brick implement `BrickSpinner.OnItemSelectedListener<PickableMusicalInstrument>`